docs/installation.md

```diff
@@ -0,0 +1,35 @@
+# Install an epoch node using a release binary
+
+This document describes how to install an epoch node using a release binary.
+
+In order to install an epoch node using a release binary, you need to:
+* Install the required dependencies;
+* Deploy the node.
+
+## Install dependencies
+
+### Ubuntu 16.04 package
+
+The Ubuntu 16.04 package requires a libsodium shared library (v1.0.16) in `/usr/local/lib/libsodium.so.23` (`wget https://download.libsodium.org/libsodium/releases/libsodium-1.0.16.tar.gz`, unpack, then do `./configure && make && sudo make install && sudo ldconfig`).
+
+### macOS package
+
+The macOS package has:
+* A hard dependency on OpenSSL v1.0.0 installed with [Homebrew](https://brew.sh/) in its default path `/usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib`;
+* A hard dependency on libsodium v1.0.16 installed with [Homebrew](https://brew.sh/) in its default path `/usr/local/opt/libsodium/lib/libsodium.23.dylib`.
+
+In case you have installed either of them in a non-default path, you could use symlink(s) to work around the issue.
+
+## Deploy node
+
+In the instructions below, the node is deployed in directory `/tmp/node`: you may prefer to deploy the node in an alternative (and less ephemeral) location - e.g. a `node` directory inside your home directory - by amending the instructions accordingly.
+It is recommended that the partition where the node directory is has at least 10 GB free: this is needed for the chain and the log files.
+
+Open a Terminal window or get to the command line.
+
+Create a directory and unpack the downloaded package (you may need to amend the directory and/or file name of the package):
+```bash
+mkdir /tmp/node
+cd /tmp/node
+tar xf ~/Downloads/epoch-0.9.0-osx-10.12.6.tar.gz
```

Comment:
we can use wildcard here `tar xf ~/Downloads/epoch-*.tar.gz` ?

Reply:
What if user has multiple archives there? I do not like hardcoding package version and osx version, though (mitigated with "amend" comment in parentheses) I think it is still a better user experience.

I think potential better alternative would be something like "Let us assume you downloaded release binary and it has absolute path `RELEASE_ARCHIVE` ..."